front/pages/w/[wId]/u/chat/[cId]/index.tsx

```diff
@@ -496,11 +523,12 @@ const COMMANDS: { cmd: string; description: string }[] = [
   {
     cmd: "/follow-up",
     description:
-      "Follow-up with the assistant without performing a document retrieval",
+      "Forces the assistant to answer *whithout* querying the data sources",
   },
   {
     cmd: "/retrieve",
```

Comment:
I think we either:
- Remove this one
- Or if someone does `/retrieve ...` we don't go to OAI function and retrieve with that exact query no assistant model. Would that work?

Reply:
I would vote for the later but not strongly held.